README.md

```diff
@@ -7,6 +7,17 @@ A bunch of the code, ideas, and links come from the OMC team's work week Hackath
 ```bash
 npm run dev
 ```
+## Setting local environment variables
```

Comment:
This section needs to be before the "Running the development server" section, I think...